core/src/main/java/com/alibaba/nacos/core/file/WatchFileCenter.java

```diff
@@ -154,9 +154,8 @@ public WatchDirJob(String paths) throws NacosException {
 			}
 
 			this.callBackExecutor = ExecutorFactory
-					.newFixExecutorService(WatchFileCenter.class.getCanonicalName(),
-							1,
-							new NameThreadFactory("com.alibaba.nacos.file.watch-" + paths));
+                    .newSingleExecutorService(
```

Comment:
Should use Managed. newSingleExecutorService?

Reply:
Because this ExecutorService belong to WatchDirJob instance. I worry about the WatchFileCenter will dymaic add/remove WatchDirJob instance .If exist remove behavior ,it should manage itself

Reply:
@chuntaojun what do you think

Reply:
yes, it should be manage itself

Reply:
ok